src/server/nuclearnet/fake_nuclearnet_client.ts

```diff
@@ -29,8 +36,9 @@ export class FakeNUClearNetClient implements NUClearNetClient {
   public connect(options: NUClearNetOptions): () => void {
     this.peer = {
       name: options.name,
-      address: '127.0.0.1',
-      port: 7447,
+      // Address and port are not used for anything yet, dummy values used.
+      address: 'fake_address',
+      port: 0,
```

Comment:
Well any Uint16 is valid so if it gets in one of those it's all over :P 
Wonder what happens when you cast NaN to a unit16